src/HasManyWithInverseRelationship.php

```diff
@@ -8,15 +8,23 @@
 
 class HasManyWithInverseRelationship extends HasMany
 {
-    protected string $relationToParent;
+    /**
+     * @var string
+     */
+    protected $relationToParent;
 
-    public function __construct(Builder $query, Model $parent, $foreignKey, $localKey, $relationToParent)
+    public function __construct(Builder $query, Model $parent, string $foreignKey, string $localKey, string $relationToParent)
```

Comment:
I originally didn't want to add typehints here because the parent class doesn't use them, but seems like the docblocks specify those types as `string` so I'll accept this.

Reply:
Okay!